extension/scripts/content-script.js

```diff
@@ -608,6 +609,12 @@ function get_fixture(fixtureID, teamID){
     }
     return "Blank"
 }
+function get_player_event_data(gameweek, playerID){
+
+    let all_players = LAST_FEW_EVENTS_DATA[gameweek]["elements"];
+    return all_players.find((player)=>{return player.id == playerID});
+}
+
```

Comment:
This function uses the find method to return the player data for the provided id.